src/universal/redux/makeReducer.js

```diff
@@ -35,6 +36,7 @@ const appReducers = {
   [DEFAULT_AUTH_REDUCER_NAME]: auth,
   cashay: cashayReducer,
   form: formReducer.plugin(formPlugin),
+  notifications: notificationReducer,
```

Comment:
let's refactor to 'dashboard' or something similar so we don't confuse the notification bar with notifications